client/my-sites/checkout/checkout-thank-you/domains/style.scss

```diff
@@ -2,26 +2,89 @@
 @import '@wordpress/base-styles/breakpoints';
 @import '@wordpress/base-styles/mixins';
 
-.is-section-checkout-thank-you .layout__content {
-    padding-right: 0;
-    padding-left: 0;
+.is-section-checkout-thank-you {
+	background: var( --studio-white );
+	
+	.layout__content {
+		padding-right: 0;
+		padding-left: 0;
+	}
 }
 
 .checkout-thank-you__domains {
+	&.thank-you__container {
+		
+		margin-top: calc( -79px + var( --masterbar-height ) );
+	
+		@include breakpoint-deprecated( '<960px' ) {
+			margin-top: calc( -71px + var( --masterbar-height ) );
+		}
+	
+		@include breakpoint-deprecated( '<660px' ) {
+			margin-top: -1px;
+		}
+	}
+
 	.thank-you {
 		&__body {
+			@include break-zoomed-in {
+				margin-top: 32px;
+			}
+
+			@include break-small {
+				margin-top: 50px;
+			}
+
 			div {
 				min-width: unset;
 			}
 
 			.domain-mapping-thank-you__button {
 				width: 144px;
 			}
+
+			& > div {
+				padding: 0 24px;
+			}
+		}
+
+		&__step {
+			padding-bottom: 24px;
+			align-items: center;
+
+			div > p {
+				@include break-mobile {
+					margin-bottom: 0;
+				}
+			}
+
+			&-cta {
+				margin-top: 0;
+
+				@include break-zoomed-in {
+					width: 100%;
+
+					.domain-mapping-thank-you__button {
+						margin-top: 4px;
+						width: 100%;
+					}
+				}
+				
+				@include break-mobile {
+					width: 144px;
+
+					.domain-mapping-thank-you__button {
+						margin-top: 0;
+						width: 144px;
+					}
+				}
+			}
 		}
+
 		&__container-header {
 			width: auto;
 			background-color: var( --studio-blue-50 );
-			padding: 50px 20px 30px;
+			padding: 0 20px;
```

Comment:
How many of these styles are actually more general things that we should be fixing for all of the post-checkout pages? I am sure we have a bunch of issues with the CSS due to us targeting a more "normal" Calypso context/frame, but all of these CSS tweaks feel like they should most likely be moved into the main ThankYou CSS, probably behind the `.is-section-checkout-thank-you` top-level selector.